From 9d6c51b2118b24cd94f3e2951ff0e9ee82afbd98 Mon Sep 17 00:00:00 2001
Message-Id: <9d6c51b2118b24cd94f3e2951ff0e9ee82afbd98.1555911520.git.liu.denton@gmail.com>
In-Reply-To: <cover.1555487380.git.liu.denton@gmail.com>
References: <cover.1555487380.git.liu.denton@gmail.com>
From: Denton Liu <liu.denton@gmail.com>
Date: Sun, 21 Apr 2019 22:27:50 -0700
Subject: [PATCH] cache.h: fix mismerge of 'dl/no-extern-in-func-decl'
To: Git Mailing List <git@vger.kernel.org>,
    Junio C Hamano <gitster@pobox.com>
Cc: Git Mailing List <git@vger.kernel.org

After merging 'dl/no-extern-in-func-decl' into 'pu', it seems like the
conflict resolution mostly went well except for one spot, which this
patch should fix. Although this is only currently only a problem in
'pu', we should fix the entry rerere cache for if this branch ends up
merged in 'next' in the future.

Signed-off-by: Denton Liu <liu.denton@gmail.com>
---

I guess while we're at it, I noticed that these commits are all missing
signoffs from you. We should probably attach those while we're fixing
this up.

Thanks,

Denton

 cache.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cache.h b/cache.h
index 5f8607d4f4..bb53660f45 100644
--- a/cache.h
+++ b/cache.h
@@ -1393,7 +1393,7 @@ int repo_get_oid_mb(struct repository *r, const char *str, struct object_id *oid
 void maybe_die_on_misspelt_object_name(struct repository *repo,
 				       const char *name,
 				       const char *prefix);
-extern enum get_oid_result get_oid_with_context(struct repository *repo, const char *str,
+enum get_oid_result get_oid_with_context(struct repository *repo, const char *str,
 				unsigned flags, struct object_id *oid,
 				struct object_context *oc);
 
-- 
2.21.0.1000.g11cd861522

